### DISCUSSION TITLE
Feature - Existing chats open to the most recent message

### DISCUSSION THREAD

**Discussion by as1164** [Taewan-P/gpt_mobile#59] [Ideas]
First off, I love the design of this app. It's very simple and easy to use. Been using it for a day and have encountered no major issues.

One feature I think would be good is when an old chat is opened, it opens to the most recent part of the conversation instead of the beginning. It's convenient when you want to continue a prior conversation. The ChatGPT browser app also does this. Thanks

**Comment by Taewan-P** [comment]
This is now implemented in #66
Will be in the next release.
Thanks!
